MyProject.py

```diff
@@ -52,17 +56,17 @@ def build():
 if __name__ == "__main__":
 
     compress_process = multiprocessing.Process(target=compress)
-    build_process = multiprocessing.Process(target=build)
+    # build_process = multiprocessing.Process(target=build)
 
     compress_process.start()
-    build_process.start()
+    # build_process.start()
 
     compress_process.join()
-    #analyzer.analyze()
-    build_process.join()
-
+    # analyzer.analyze()
```

Comment:
Only changes in these 3 lines are needed, and please stick `analyzer.detect_boundary()` before `build_process.join()` because we don't have to start detect after build process complete